data_loss.rst

```diff
@@ -473,3 +474,30 @@ The following command can be used to monitor the progress of any ongoing balance
 .. code-block:: console
 
     btrfs balance status /mnt2/pool-name
+
+.. _rockonrestore:
+
+Rock-on Restore
+^^^^^^^^^^^^^^^
+
+When Rock-ons are not visible after data restore it is possible that they are either running silently in the background
+or Rockstor does not have the needed information to restore the Rock-ons. Here is how to get your Rock-ons back up and
+running.
+
+Rock-on data was not stored on system drive (recommended)
```

Comment:
```suggestion
Rock-on associated shares are not stored on the system drive (recommended)
```

This way we keep with the Web-UI reference of shares, and cover both the rock-ons-root and the specific Rock-on config/data shares encouraged for each Rock-on.